source/store/actions/CaseActions.js

```diff
@@ -27,11 +36,36 @@ export async function updateCase(
     answers: convertAnswersToArray(answerObject, formQuestions),
     currentPosition,
     currentFormId: formId,
-    encryption: { type: 'decrypted' },
+    encryption,
   };
 
+  const [encryptionStatus, updatedForm] = await updateFormEncryption(
+    user.personalNumber,
+    updateCaseRequestBody
+  );
+
+  updateCaseRequestBody = updatedForm;
+
   if (encryptAnswers) {
-    updateCaseRequestBody = await encryptFormAnswers(user, updateCaseRequestBody);
+    let encryptedAnswers;
+    try {
+      encryptedAnswers = await encryptFormAnswers(user.personalNumber, updateCaseRequestBody);
+    } catch (error) {
+      if (error?.status === 'missingSymmetricKey') {
+        // Symmetric key (co-applicant) is desired but no key has been generated yet,
+        // for now, encrypt with private AES key.
+        console.log('Missing symmetric key - using private AES key for now');
```

Comment:
Wasn't intended to. Are you referring to how it might sound like a "todo"? I have probably worded this poorly then - this is intended behavior 🙂. This simply states that this is the code path where no symmetric key is available yet, which is a possible user scenario.

I will go through all the body comments again and attempt to replace the comments with better variable naming as I go.